src/Api/Model/Languageforge/Lexicon/Command/LexEntryCommands.php

```diff
@@ -55,6 +55,7 @@ public static function updateEntry($projectId, $params, $userId, $mergeQueuePath
         $project->lastEntryModifiedDate = $now;
         if (array_key_exists('id', $params) && $params['id'] != '') {
             $entry = new LexEntryModel($project, $params['id']);
+            $oldEntry = new LexEntryModel($project, $params['id']); // NOT $entry = $oldEntry
```

Comment:
@rmunn do you know about PHP object cloning? e.g. `$entry = clone $oldEntry;`
See http://php.net/manual/en/language.oop5.cloning.php